Datagrid/Datagrid.php

```diff
@@ -126,15 +126,15 @@ public function buildPager()
             $this->formBuilder->add($filter->getFormName(), $type, $options);
         }
 
-        $this->formBuilder->add('_sort_by', 'hidden');
+        $this->formBuilder->add('_sort_by', 'Symfony\Component\Form\Extension\Core\Type\HiddenType');
```

Comment:
@WouterJ Stop me if I didn't understand but `$this->formBuilder` is a Symfony `FormBuilderInterface` instance.

And this instance does not accept class on SF `2.3` AFAIK.